Integrations/PaloAltoNetworks_Traps/PaloAltoNetworks_Traps.py

```diff
@@ -0,0 +1,550 @@
+''' IMPORTS '''
+
+import demistomock as demisto
+from CommonServerPython import *
```

Comment:
I think all 3 lines of the import should be moved to the beginning of the file. That is where we replace it with empty string.

Reply:
Do you mean this lines?
```
import demistomock as demisto
from CommonServerPython import *
from CommonServerUserPython import *
```
If so, fixed.